### DISCUSSION TITLE
Thermostat with condition

### DISCUSSION THREAD

**Discussion by Enjoy-Combi** [ScratMan/HASmartThermostat#65] [Ideas]
Hello,

I am looking for a thermostat taking into account a temperature as condition to heat or not. This because it would be used for solar panel for swimming pool. Heating has to be granted only if solar panel temperature is above swimming pool temperature, if not the swimming pool with be cooled down :-)

Is it something which could be added to the thermostat ?

**Comment by ScratMan** [comment]
Hello, the thermostat operating mode can be set by Home Assistant automatically either heat or off based on the trigger and/or condition with automation.

**Reply by Enjoy-Combi** [reply]
So simple. I feel stupid now :-)
